src/tools/msc.lua

```diff
@@ -386,6 +386,13 @@
 			table.insert(flags, '/NODEFAULTLIB:' .. ignore)
 		end
 
+		-- If the type is ConsoleApp, WindowedApp, or SharedLib and PROFILE is set, add the appropriate flag.
```

Comment:
I find that kind of comment useless, it just repeats the below code (which is clear).

Reply:
Yeah, I wrote the comment for myself before writing all the code. Will remove.